### ISSUE TITLE
Improve lightbox accessibility

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#329]
<strong>Every input concering lightbox is deleted on Github.
#310 #212 and #256.</strong>

Yes, it's difficult to make a lightbox accessible.
But please implement the things "access for all" mentioned:

The first and last step are quite easy to implement:

1) Annonce the lightbox with a hidden remark in the link
==> Use "opens a lightbox"

6) Complete the overlay DIV-Element with the role='document', like this a screen reader read the content of the lightbox

The other steps are:

2) The focus should be automatically in the beginning of the lightbox.

3) Without closing the lightbox the focus should stay there - aria-hidden='true' for the whole page

4 Close-Link works in tab mode, but not in browse mode. Esc ist ok.

5) After closing the lightbox a screen reader should read further from the position of the lightbox (this will be importent in example pages)

**Comment by desobryc** [comment]
We still have the problem, at least in the cms, that the alt-tags of lightboxes are empty / cannot be edited. It is to be defined if this is a webguidelines problem or only a problem of the implementation in the cms.
